mon_school/public/js/sketch_like.js

```diff
@@ -0,0 +1,54 @@
+var notLikedLikeIconSrc = "/assets/school/icons/like.svg";
+var likedLikeIconSrc = "/assets/mon_school/icons/red-like.svg";
+
+function toggleLike($btn) {
+  if (!frappe.user_id || (frappe.user && !frappe.user.name)) {
```

Comment:
I don't think `frappe` object has `user_id` or `user` defined. This condition is True always and not allowing me to like a sketch even after logging in.

You should check for `frappe.session.user`.

Reply:
Ah okay. For me frappe.session.user didn't exist so I found frappe.user_id and frappe.user by Googling. I tried it in browser console, sometimes one existed and the other didn't hence this hacky condition.

This looks like a fix though: https://discuss.erpnext.com/t/get-user-id-in-javascript/48570/9?u=kaustubh

Reply:
I've used 
```
$(function() {
...
});
```

and that worked for me. Not sure how different it is from `$(document).ready`.